<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="4" total="9" passed="5">
  <reporter-output>
  </reporter-output>
  <suite name="Suite" duration-ms="24619" started-at="2016-04-23T04:37:09Z" finished-at="2016-04-23T04:37:34Z">
    <groups>
    </groups>
    <test name="Login" duration-ms="24619" started-at="2016-04-23T04:37:09Z" finished-at="2016-04-23T04:37:34Z">
      <class name="com.vsii.tsc.guru.testcase.Login">
        <test-method status="PASS" signature="setupClass()[pri:0, instance:com.vsii.tsc.guru.testcase.Login@5ebec15]" name="setupClass" is-config="true" duration-ms="29" started-at="2016-04-23T11:37:09Z" finished-at="2016-04-23T11:37:09Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setupClass -->
        <test-method status="FAIL" signature="LO01()[pri:0, instance:com.vsii.tsc.guru.testcase.Login@5ebec15]" name="LO01" duration-ms="111" started-at="2016-04-23T11:37:09Z" description="Verify Title of Login Page" finished-at="2016-04-23T11:37:09Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [true] but found [false]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:496)
	at org.testng.Assert.assertTrue(Assert.java:42)
	at org.testng.Assert.assertTrue(Assert.java:52)
	at com.vsii.tsc.guru.testcase.Login.LO01(Login.java:31)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:483)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:816)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1124)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:774)
	at org.testng.TestRunner.run(TestRunner.java:624)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:359)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:354)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:312)
	at org.testng.SuiteRunner.run(SuiteRunner.java:261)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1215)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
	at org.testng.TestNG.run(TestNG.java:1048)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:112)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:205)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:176)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- LO01 -->
        <test-method status="PASS" signature="afterMethod(org.testng.ITestResult)[pri:0, instance:com.vsii.tsc.guru.testcase.Login@5ebec15]" name="afterMethod" is-config="true" duration-ms="2324" started-at="2016-04-23T11:37:09Z" finished-at="2016-04-23T11:37:12Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=LO01 status=FAILURE method=Login.LO01()[pri:0, instance:com.vsii.tsc.guru.testcase.Login@5ebec15] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterMethod -->
        <test-method status="PASS" signature="LO02(java.lang.String, java.lang.String, java.lang.String)[pri:1, instance:com.vsii.tsc.guru.testcase.Login@5ebec15]" name="LO02" duration-ms="2295" started-at="2016-04-23T11:37:13Z" description="verify Login" data-provider="dpLogin" finished-at="2016-04-23T11:37:15Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[Tester01]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[nAgezam ]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[User or Password is not valid]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- LO02 -->
        <test-method status="PASS" signature="afterMethod(org.testng.ITestResult)[pri:0, instance:com.vsii.tsc.guru.testcase.Login@5ebec15]" name="afterMethod" is-config="true" duration-ms="665" started-at="2016-04-23T11:37:15Z" finished-at="2016-04-23T11:37:16Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=LO02 status=SUCCESS method=Login.LO02(java.lang.String, java.lang.String, java.lang.String)[pri:1, instance:com.vsii.tsc.guru.testcase.Login@5ebec15] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterMethod -->
        <test-method status="PASS" signature="LO02(java.lang.String, java.lang.String, java.lang.String)[pri:1, instance:com.vsii.tsc.guru.testcase.Login@5ebec15]" name="LO02" duration-ms="1944" started-at="2016-04-23T11:37:16Z" description="verify Login" data-provider="dpLogin" finished-at="2016-04-23T11:37:17Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[mngr22018]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[123456]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[User or Password is not valid]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- LO02 -->
        <test-method status="PASS" signature="afterMethod(org.testng.ITestResult)[pri:0, instance:com.vsii.tsc.guru.testcase.Login@5ebec15]" name="afterMethod" is-config="true" duration-ms="520" started-at="2016-04-23T11:37:17Z" finished-at="2016-04-23T11:37:18Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=LO02 status=SUCCESS method=Login.LO02(java.lang.String, java.lang.String, java.lang.String)[pri:1, instance:com.vsii.tsc.guru.testcase.Login@5ebec15] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterMethod -->
        <test-method status="PASS" signature="LO02(java.lang.String, java.lang.String, java.lang.String)[pri:1, instance:com.vsii.tsc.guru.testcase.Login@5ebec15]" name="LO02" duration-ms="2026" started-at="2016-04-23T11:37:18Z" description="verify Login" data-provider="dpLogin" finished-at="2016-04-23T11:37:20Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[Tester02]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[123456]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[User or Password is not valid]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- LO02 -->
        <test-method status="PASS" signature="afterMethod(org.testng.ITestResult)[pri:0, instance:com.vsii.tsc.guru.testcase.Login@5ebec15]" name="afterMethod" is-config="true" duration-ms="380" started-at="2016-04-23T11:37:20Z" finished-at="2016-04-23T11:37:20Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=LO02 status=SUCCESS method=Login.LO02(java.lang.String, java.lang.String, java.lang.String)[pri:1, instance:com.vsii.tsc.guru.testcase.Login@5ebec15] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterMethod -->
        <test-method status="FAIL" signature="LO02(java.lang.String, java.lang.String, java.lang.String)[pri:1, instance:com.vsii.tsc.guru.testcase.Login@5ebec15]" name="LO02" duration-ms="2580" started-at="2016-04-23T11:37:20Z" description="verify Login" data-provider="dpLogin" finished-at="2016-04-23T11:37:23Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[nAgezam ]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[User-ID must not be blank]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [true] but found [false]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:496)
	at org.testng.Assert.assertTrue(Assert.java:42)
	at org.testng.Assert.assertTrue(Assert.java:52)
	at com.vsii.tsc.guru.testcase.Login.LO02(Login.java:44)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:483)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:816)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1124)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:774)
	at org.testng.TestRunner.run(TestRunner.java:624)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:359)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:354)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:312)
	at org.testng.SuiteRunner.run(SuiteRunner.java:261)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1215)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
	at org.testng.TestNG.run(TestNG.java:1048)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:112)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:205)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:176)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- LO02 -->
        <test-method status="PASS" signature="afterMethod(org.testng.ITestResult)[pri:0, instance:com.vsii.tsc.guru.testcase.Login@5ebec15]" name="afterMethod" is-config="true" duration-ms="372" started-at="2016-04-23T11:37:23Z" finished-at="2016-04-23T11:37:23Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=LO02 status=FAILURE method=Login.LO02(java.lang.String, java.lang.String, java.lang.String)[pri:1, instance:com.vsii.tsc.guru.testcase.Login@5ebec15] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterMethod -->
        <test-method status="FAIL" signature="LO02(java.lang.String, java.lang.String, java.lang.String)[pri:1, instance:com.vsii.tsc.guru.testcase.Login@5ebec15]" name="LO02" duration-ms="921" started-at="2016-04-23T11:37:23Z" description="verify Login" data-provider="dpLogin" finished-at="2016-04-23T11:37:24Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[mngr22018]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[Password must not be blank]]>
              </value>
            </param>
          </params>
          <exception class="org.openqa.selenium.NoSuchElementException">
            <message>
              <![CDATA[Unable to locate element: {"method":"name","selector":"uid"}
Command duration or timeout: 918 milliseconds
For documentation on this error, please visit: http://seleniumhq.org/exceptions/no_such_element.html
Build info: version: '2.52.0', revision: '4c2593cfc3689a7fcd7be52549167e5ccc93ad28', time: '2016-02-11 11:22:43'
System info: host: 'trangntt-w7', ip: '192.168.100.30', os.name: 'Windows 7', os.arch: 'amd64', os.version: '6.1', java.version: '1.8.0_25'
Driver info: org.openqa.selenium.firefox.FirefoxDriver
Capabilities [{applicationCacheEnabled=true, rotatable=false, handlesAlerts=true, databaseEnabled=true, version=45.0.2, platform=WINDOWS, nativeEvents=false, acceptSslCerts=true, webStorageEnabled=true, locationContextEnabled=true, browserName=firefox, takesScreenshot=true, javascriptEnabled=true, cssSelectorsEnabled=true}]
Session ID: 22f9f18a-454e-474d-b0d9-91948f46b75c
*** Element info: {Using=name, value=uid}]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.openqa.selenium.NoSuchElementException: Unable to locate element: {"method":"name","selector":"uid"}
Command duration or timeout: 918 milliseconds
For documentation on this error, please visit: http://seleniumhq.org/exceptions/no_such_element.html
Build info: version: '2.52.0', revision: '4c2593cfc3689a7fcd7be52549167e5ccc93ad28', time: '2016-02-11 11:22:43'
System info: host: 'trangntt-w7', ip: '192.168.100.30', os.name: 'Windows 7', os.arch: 'amd64', os.version: '6.1', java.version: '1.8.0_25'
Driver info: org.openqa.selenium.firefox.FirefoxDriver
Capabilities [{applicationCacheEnabled=true, rotatable=false, handlesAlerts=true, databaseEnabled=true, version=45.0.2, platform=WINDOWS, nativeEvents=false, acceptSslCerts=true, webStorageEnabled=true, locationContextEnabled=true, browserName=firefox, takesScreenshot=true, javascriptEnabled=true, cssSelectorsEnabled=true}]
Session ID: 22f9f18a-454e-474d-b0d9-91948f46b75c
*** Element info: {Using=name, value=uid}
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:408)
	at org.openqa.selenium.remote.ErrorHandler.createThrowable(ErrorHandler.java:206)
	at org.openqa.selenium.remote.ErrorHandler.throwIfResponseFailed(ErrorHandler.java:158)
	at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:678)
	at org.openqa.selenium.remote.RemoteWebDriver.findElement(RemoteWebDriver.java:363)
	at org.openqa.selenium.remote.RemoteWebDriver.findElementByName(RemoteWebDriver.java:461)
	at org.openqa.selenium.By$ByName.findElement(By.java:303)
	at org.openqa.selenium.remote.RemoteWebDriver.findElement(RemoteWebDriver.java:355)
	at org.openqa.selenium.support.pagefactory.DefaultElementLocator.findElement(DefaultElementLocator.java:69)
	at org.openqa.selenium.support.pagefactory.internal.LocatingElementHandler.invoke(LocatingElementHandler.java:38)
	at com.sun.proxy.$Proxy9.sendKeys(Unknown Source)
	at com.vsii.tsc.guru.pages.method.LoginPageMethod.loginToManagerPage(LoginPageMethod.java:118)
	at com.vsii.tsc.guru.testcase.Login.LO02(Login.java:38)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:483)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:816)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1124)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:774)
	at org.testng.TestRunner.run(TestRunner.java:624)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:359)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:354)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:312)
	at org.testng.SuiteRunner.run(SuiteRunner.java:261)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1215)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
	at org.testng.TestNG.run(TestNG.java:1048)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:112)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:205)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:176)
Caused by: org.openqa.selenium.NoSuchElementException: Unable to locate element: {"method":"name","selector":"uid"}
For documentation on this error, please visit: http://seleniumhq.org/exceptions/no_such_element.html
Build info: version: '2.52.0', revision: '4c2593cfc3689a7fcd7be52549167e5ccc93ad28', time: '2016-02-11 11:22:43'
System info: host: 'trangntt-w7', ip: '192.168.100.30', os.name: 'Windows 7', os.arch: 'amd64', os.version: '6.1', java.version: '1.8.0_25'
Driver info: driver.version: unknown
	at <anonymous class>.FirefoxDriver.prototype.findElementInternal_(file:///C:/Users/trangntt/AppData/Local/Temp/anonymous4667028526666105824webdriver-profile/extensions/fxdriver@googlecode.com/components/driver-component.js:10723)
	at <anonymous class>.FirefoxDriver.prototype.findElement(file:///C:/Users/trangntt/AppData/Local/Temp/anonymous4667028526666105824webdriver-profile/extensions/fxdriver@googlecode.com/components/driver-component.js:10732)
	at <anonymous class>.DelayedCommand.prototype.executeInternal_/h(file:///C:/Users/trangntt/AppData/Local/Temp/anonymous4667028526666105824webdriver-profile/extensions/fxdriver@googlecode.com/components/command-processor.js:12614)
	at <anonymous class>.DelayedCommand.prototype.executeInternal_(file:///C:/Users/trangntt/AppData/Local/Temp/anonymous4667028526666105824webdriver-profile/extensions/fxdriver@googlecode.com/components/command-processor.js:12619)
	at <anonymous class>.DelayedCommand.prototype.execute/<(file:///C:/Users/trangntt/AppData/Local/Temp/anonymous4667028526666105824webdriver-profile/extensions/fxdriver@googlecode.com/components/command-processor.js:12561)
]]>
            </full-stacktrace>
          </exception> <!-- org.openqa.selenium.NoSuchElementException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- LO02 -->
        <test-method status="PASS" signature="afterMethod(org.testng.ITestResult)[pri:0, instance:com.vsii.tsc.guru.testcase.Login@5ebec15]" name="afterMethod" is-config="true" duration-ms="2588" started-at="2016-04-23T11:37:24Z" finished-at="2016-04-23T11:37:27Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=LO02 status=FAILURE method=Login.LO02(java.lang.String, java.lang.String, java.lang.String)[pri:1, instance:com.vsii.tsc.guru.testcase.Login@5ebec15] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterMethod -->
        <test-method status="PASS" signature="LO02(java.lang.String, java.lang.String, java.lang.String)[pri:1, instance:com.vsii.tsc.guru.testcase.Login@5ebec15]" name="LO02" duration-ms="1068" started-at="2016-04-23T11:37:27Z" description="verify Login" data-provider="dpLogin" finished-at="2016-04-23T11:37:28Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[ ]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[ ]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[User or Password is not valid]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- LO02 -->
        <test-method status="PASS" signature="afterMethod(org.testng.ITestResult)[pri:0, instance:com.vsii.tsc.guru.testcase.Login@5ebec15]" name="afterMethod" is-config="true" duration-ms="250" started-at="2016-04-23T11:37:28Z" finished-at="2016-04-23T11:37:28Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=LO02 status=SUCCESS method=Login.LO02(java.lang.String, java.lang.String, java.lang.String)[pri:1, instance:com.vsii.tsc.guru.testcase.Login@5ebec15] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterMethod -->
        <test-method status="FAIL" signature="LO02(java.lang.String, java.lang.String, java.lang.String)[pri:1, instance:com.vsii.tsc.guru.testcase.Login@5ebec15]" name="LO02" duration-ms="1897" started-at="2016-04-23T11:37:28Z" description="verify Login" data-provider="dpLogin" finished-at="2016-04-23T11:37:30Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[mngr22018]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[nAgezam ]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[Manger Id : mngr22018]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [true] but found [false]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:496)
	at org.testng.Assert.assertTrue(Assert.java:42)
	at org.testng.Assert.assertTrue(Assert.java:52)
	at com.vsii.tsc.guru.testcase.Login.LO02(Login.java:44)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:483)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:816)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1124)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:774)
	at org.testng.TestRunner.run(TestRunner.java:624)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:359)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:354)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:312)
	at org.testng.SuiteRunner.run(SuiteRunner.java:261)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1215)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
	at org.testng.TestNG.run(TestNG.java:1048)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:112)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:205)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:176)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- LO02 -->
        <test-method status="PASS" signature="afterMethod(org.testng.ITestResult)[pri:0, instance:com.vsii.tsc.guru.testcase.Login@5ebec15]" name="afterMethod" is-config="true" duration-ms="361" started-at="2016-04-23T11:37:30Z" finished-at="2016-04-23T11:37:30Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=LO02 status=FAILURE method=Login.LO02(java.lang.String, java.lang.String, java.lang.String)[pri:1, instance:com.vsii.tsc.guru.testcase.Login@5ebec15] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterMethod -->
        <test-method status="PASS" signature="LO03(java.lang.String, java.lang.String)[pri:1, instance:com.vsii.tsc.guru.testcase.Login@5ebec15]" name="LO03" duration-ms="978" started-at="2016-04-23T11:37:30Z" description="verify_Reset_Button" data-provider="reset" finished-at="2016-04-23T11:37:31Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[abc]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[abc123]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- LO03 -->
        <test-method status="PASS" signature="afterMethod(org.testng.ITestResult)[pri:0, instance:com.vsii.tsc.guru.testcase.Login@5ebec15]" name="afterMethod" is-config="true" duration-ms="2246" started-at="2016-04-23T11:37:31Z" finished-at="2016-04-23T11:37:34Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=LO03 status=SUCCESS method=Login.LO03(java.lang.String, java.lang.String)[pri:1, instance:com.vsii.tsc.guru.testcase.Login@5ebec15] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterMethod -->
        <test-method status="PASS" signature="teardownClass()[pri:0, instance:com.vsii.tsc.guru.testcase.Login@5ebec15]" name="teardownClass" is-config="true" duration-ms="1" started-at="2016-04-23T11:37:34Z" finished-at="2016-04-23T11:37:34Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- teardownClass -->
      </class> <!-- com.vsii.tsc.guru.testcase.Login -->
      <class name="com.vsii.tsc.utility.TestBase">
        <test-method status="PASS" signature="setupSuite()[pri:0, instance:com.vsii.tsc.utility.TestBase@3d8c7aca]" name="setupSuite" is-config="true" duration-ms="8033" started-at="2016-04-23T11:37:01Z" finished-at="2016-04-23T11:37:09Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setupSuite -->
        <test-method status="PASS" signature="teardownSuite()[pri:0, instance:com.vsii.tsc.utility.TestBase@3d8c7aca]" name="teardownSuite" is-config="true" duration-ms="1" started-at="2016-04-23T11:37:34Z" finished-at="2016-04-23T11:37:34Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- teardownSuite -->
      </class> <!-- com.vsii.tsc.utility.TestBase -->
    </test> <!-- Login -->
  </suite> <!-- Suite -->
</testng-results>
